src/material-experimental/mdc-button/fab.ts

```diff
@@ -163,6 +165,8 @@ export class MatMiniFabButton extends MatButtonBase {
     // Add a class that applies to all buttons. This makes it easier to target if somebody
     // wants to target all Material buttons.
     '[class.mat-mdc-button-base]': 'true',
+    '(focus)': '_focused = true',
+    '(blur)': '_focused = false',
```

Comment:
This will introduce two new change detections on every interaction with the button for a feature that most users likely won't be using. Couldn't the same be achieved with a `:focus` selector or some of the `FocusMonitor` classes like we had before?

Reply:
> This will introduce two new change detections on every interaction with the button for a feature that most users likely won't be using. 

Yeah, this was my biggest concern as well. I don't know how expensive these CDs are.

> Couldn't the same be achieved with a `:focus` selector or some of the `FocusMonitor` classes like we had before?

Sorry, do you mind clarifying your suggestion? It might help to look at the two points of motivation in the CL description and ensure that the recommendation addresses them. Also happy to chat over Slack if that's easier.